app/views/shared/_footer_lite.html.erb

```diff
@@ -0,0 +1,70 @@
+<% show_language_dropdown = I18n.available_locales.count > 1
+sanitized_requested_url = request.query_parameters.slice(:request_id) %>
+
+<footer class='footer bg-light-blue sm-bg-navy'>
+  <% if show_language_dropdown %>
+    <div class='sm-hide border-bottom'>
+      <div class='container cntnr-wide py1 px2 lg-px0 h5'>
+        <div class='i18n-mobile-toggle center'>
+          <%= link_to('#', class: 'block text-decoration-none blue fs-13p', 'aria-expanded': 'false') do %>
+            <%= image_tag asset_url('globe-blue.svg'), width: 12, class: 'mr1', alt: '',
+              'aria-hidden': 'true' %><%= t('i18n.language') %><span class='caret inline-block ml-tiny' aria-hidden='true'>&#9662;</span>
+          <% end %>
+        </div>
+      </div>
+    </div>
+    <div class='i18n-mobile-dropdown sm-hide display-none'>
+      <ul class='list-reset mb0 white center'>
+        <%  I18n.available_locales.each do |locale| %>
+          <li class='border-bottom'>
+            <%= link_to t("i18n.locale.#{locale}"), sanitized_requested_url.merge(locale: locale), class: 'block py-12p px2 text-decoration-none blue fs-13p'%>
+          </li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+  <div class='container py1 px2 lg-px0 <%= 'sm-py0' if show_language_dropdown %>'>
+    <div class='flex flex-center'>
+      <div class='flex flex-auto flex-first'>
+        <%= link_to('https://gsa.gov', class: 'flex flex-center text-decoration-none white h6', target: '_blank', 'aria-label': t('shared.footer_lite.gsa')) do %>
```

Comment:
The text associated with this link doesn't appear on mobile due to `sm-show`, so I added an `aria-label` and now screen reader announces link as "US General Services Administration" instead of reading out the URL.

Commit diff of accessibility change: https://github.com/18F/identity-idp/pull/4000/commits/ff4e7e5e8677b792251e3735cdf8fddb8cf85333